index.js

```diff
@@ -1,8 +1,11 @@
 const http = require('http');
 const axios = require('axios');
 const mongoose = require('mongoose');
+var CronJob = require('cron').CronJob;
 require('dotenv').config();
 
+new CronJob('0 0 12 * * * *', () => putAllMoviePagesInDatabase(), null, true, 'America/Los_Angeles');
```

Comment:
You trying to start Cron and than save movies, but you not sure that db connection established